plugins/src/webrtcsink/imp.rs

```diff
@@ -242,22 +245,45 @@ impl Default for State {
     }
 }
 
+fn make_converter_for_video_caps(caps: &gst::Caps) -> Result<gst::Element, Error> {
+    assert!(caps.size() == 1);
```

Comment:
```suggestion
    assert_eq!(caps.size(), 1);
```

But maybe you actually want

```suggestion
    assert!(caps.is_fixed());
```

?